Bump to latest Owasp DependencyCheck

# Motivation
Errors such as: 
```
[2017-11-14 16:25:50.124] ERROR CentralAnalyzer - Could not connect to Central search. Analysis failed.
java.io.IOException: Finally failed connecting to Central search. Giving up after 5 tries.
	at org.owasp.dependencycheck.analyzer.CentralAnalyzer.fetchMavenArtifacts(CentralAnalyzer.java:288)
	at org.owasp.dependencycheck.analyzer.CentralAnalyzer.analyzeDependency(CentralAnalyzer.java:198)
	at org.owasp.dependencycheck.analyzer.AbstractAnalyzer.analyze(AbstractAnalyzer.java:137)
	at org.owasp.dependencycheck.AnalysisTask.call(AnalysisTask.java:88)
	at org.owasp.dependencycheck.AnalysisTask.call(AnalysisTask.java:37)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Could not connect to MavenCentral (400): Bad Request
	at org.owasp.dependencycheck.data.central.CentralSearch.searchSha1(CentralSearch.java:181)
	at org.owasp.dependencycheck.analyzer.CentralAnalyzer.fetchMavenArtifacts(CentralAnalyzer.java:266)
	... 8 more
```

https://github.com/jeremylong/DependencyCheck/tree/v3.0.2
https://github.com/jeremylong/DependencyCheck/issues/978

# Suggested Musical Pairing
https://soundcloud.com/wax-poetics/instant-classic-selection

--- a/project.clj
+++ b/project.clj
@@ -9,7 +9,7 @@
     [clansi "1.0.0"]
     [org.clojure/data.json "0.2.6"]
     [org.slf4j/slf4j-simple "1.7.25"]
-    [org.owasp/dependency-check-core "3.0.1"]
+    [org.owasp/dependency-check-core "3.0.2"]
     [rm-hull/table "0.6.4"]]
   :scm {:url "git@github.com:rm-hull/lein-nvd.git"}
   :source-paths ["src"]

--- a/test/nvd/config_test.clj
+++ b/test/nvd/config_test.clj
@@ -26,7 +26,7 @@
    [clojure.test :refer :all]
    [nvd.config :refer :all]))
 
-(def dependency-check-version "3.0.1")
+(def dependency-check-version "3.0.2")
 
 (deftest check-app-name
   (is (= "unknown" (app-name {:nome "hello-world" :version "0.0.1"})))

